### PR TITLE
Task/revpi 2585/error handling

### PR DESCRIPTION
**Pull request by iluminat23**

### CODE REVIEW

**Review by ferdrich-kunbus — COMMENTED** [review]
One thing to consider or clarify, rest seems ok. A full source review should be done somewhen in the future (when the reviewer has some more knowledge about rust).

File: revpi-hat-eep/src/gpio.rs [hunk 1]
```diff
@@ -289,9 +289,9 @@ impl Display for GpioBank {
 }
 
 impl TryFrom<GpioBank> for gpio_map::EepAtomGpioMapData {
-    type Error = RevPiError;
+    type Error = Box<dyn std::error::Error>;
 
-    fn try_from(bank: GpioBank) -> Result<Self, Self::Error> {
+    fn try_from(bank: GpioBank) -> Result<Self, Box<dyn std::error::Error>> {
```
[the remark this answers is not in the document]
Comment (iluminat23) [reply revpi-hat-eep/src/gpio.rs:294]: Good Catch!

**Comment by ferdrich-kunbus** [comment]
Missing 't' in constraints in commit message of [bc3ab28](https://github.com/RevolutionPi/revpi-hat-eeprom/pull/13/commits/bc3ab28a1103ea93914bcb9fcdbc85c1ee1ee9f1)
Remove 'p' in 'atpom' in commit message of [f69e60a](https://github.com/RevolutionPi/revpi-hat-eeprom/pull/13/commits/f69e60afad3c5a9d3a578f950ca07c58ae118a84)

**Review by ferdrich-kunbus — APPROVED** [review]
ok, mergeable

**Pull request merged**